docs/guide/api-vite-environment.md

```diff
@@ -434,8 +434,8 @@ function createWorkedEnvironment(userConfig) {
         ],
       },
       dev: {
-        createEnvironment(name, config) {
-          return createWorkerdDevEnvironment(name, config)
+        createEnvironment(name, config, setup) {
+          return createWorkerdDevEnvironment(name, config, setup)
```

Comment:
I don't like that `dev.createEnvironment` takes an extra param compared to `build.createEnvironment`, but I imagine that this will change as soon as the builder evolves. An example could be a top level watcher during build.

Reply:
`setup` is a terrible name in my opinion. It's primarily used as an action - to setup (ofc, a noun setup also exists, but I don't know if it applies here)

Reply:
Agreed. What should be call this? `context` maybe? I'm not kidding. I think we can't call this `options`.

Reply:
`context` actually sounds fine to me in this _context_

Reply:
Ok, used this for both `DevEnvironmentContext` and `CreateDevEnvironmentContext`. We can change one of them though later.